app/src/test/java/com/app/calculator/OperationPresenterTest.java

```diff
@@ -10,4 +10,8 @@
  * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
  */
 public class OperationPresenterTest {
+    @Test
+    public void pow_isCorrect() {
```

Comment:
why are you testing the java.lang.Math method? it has no relation to the OperationPresenter.